src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java

```diff
@@ -0,0 +1,60 @@
+public static final String CANCEL_SINGLE_BUILD = "  Cancelled build \"%s\".";
+public static final String EXIT_BUILD_CANCEL = "\n\nExiting \"%s\" successfully with %d builds cancelled.";
+public static final String START_BUILD = "\n\nStarting the \"%s\" step with build config \"%s\" from the project \"%s\".";
+public static final String BUILD_CANNOT_START = "\n\nExiting \"%s\" unsuccessfully; a build could not be started";
+public static final String BUILD_STARTED = "  Started build \"%s\" and waiting for build completion %s...";
+public static final String BUILD_WAIT_DEPLOY = "followed by a new deployment";
+public static final String BUILD_NO_POD = "\n\nExiting \"%s\" unsuccessfully; the build pod for build \"%s\" was not found in time.";
+public static final String BUILD_NO_CFG = "\n\nExiting \"%s\" unsuccessfully; the build config \"%s\" could not be read.";
+public static final String BUILD_VERIFYING = "  Verifying build \"%s\" and waiting for build completion %s...";
+public static final String START_WITH_PROJECT = "\n\nStarting the \"%s\" step with the project \"%s\".";
+public static final String CREATING = "  Creating a \"%s\"...";
+public static final String TYPE_NOT_SUPPORTED = "  The API resource \"%s\" is not currently supported by this step.";
+public static final String START_DEPLOY_CANCEL = "\n\nStarting the \"%s\" action for deployment config \"%s\" from the project \"%s\".";
+public static final String DEPLOY_NO_CFG = "\n\nExiting \"%s\" unsuccessfully; the deployment config \"%s\" could not be retrieved.";
+public static final String DEPLOY_CANCEL_NOOP = "\n\nExiting \"%s\" successfully; the deployment \"%s\" is not in-progress; its status is:  [%s].";
+public static final String DEPLOY_CANCEL_SUCCESS = "\n\nExiting \"%s\" successfully; the deployment \"%s\" has been cancelled.";
+public static final String DEPLOY_NO_REP = "\n\nExiting \"%s\" unsuccessfully; the latest deployment \"%s\" could not be retrieved.";
+public static final String START_DEPLOY = "\n\nStarting the \"%s\" step with deployment config \"%s\" from the project \"%s\".";
+public static final String DEPLOY_FAILED = "\n\nExiting \"%s\" unsuccessfully; deployment \"%s\" has completed with status:  [Failed].";
+public static final String DEPLOY_TIMED_OUT = "\n\nExiting \"%s\" unsuccessfully; gave up on deployment \"%s\" with status:  [%s].";
+public static final String DEPLOY_SUCCESS = "\n\nExiting \"%s\" successfully; deployment \"%s\" has completed with status:  [Complete].";
+public static final String DEPLOY_VERIFY_STATE = "  Verifying deployment state%s...";
+public static final String DEPLOY_VERIFY_REPLICAS = " and verifying the current deployment is at \"%s\" replica(s).";
+public static final String SCM_CALC = "\n\nThe \"%s\" SCM will return the last revision state stored in Jenkins for the image stream \"%s\" and tag \"%s\" from the project \"%s\".";
```

Comment:
hm.  i guess let's leave it SCM for now.  My only other idea is to say "SCM imagestream poller"